Makefile

```diff
@@ -1,31 +1,29 @@
 SRC_DIR="src/"
 SRC_FILES= $(shell find $(SRC_DIR) -name "*.php")
 
-dist/roger-q.phar: box.json.dist tools/box bin/roger-q.php $(SRC_FILES) composer.lock
+dist/roger-q.phar: box.json.dist tools/box.phar bin/roger-q.php $(SRC_FILES) composer.lock
 	composer install --optimize-autoloader --no-dev --no-suggest --quiet
-	./tools/box compile --quiet
+	./tools/box.phar compile --quiet
 
-tools/box:
-	wget --directory-prefix=tools --quiet https://github.com/humbug/box/releases/download/3.1.2/box.phar
-	mv tools/box.phar tools/box
-	chmod +x tools/box
+tools/box.phar:
+	wget --directory-prefix=tools --quiet https://github.com/humbug/box/releases/download/3.16.0/box.phar
```

Comment:
i downgraded to version 3 again as version 4 only works with php ^8.1

i think until everything is on php 8.1 we should still support 7.4.

Reply:
i verified that roger-q built on php 7.4 works runs fine in php 8.2